__tests__/app.test.js

```diff
@@ -28,34 +28,73 @@ describe('app routes', () => {
       return client.end(done);
     });
 
-    test('returns animals', async() => {
+    test('create todo', async() => {
 
       const expectation = [
```

Comment:
While checking your endpoints in Postman, I noticed something that you might have struggled with when trying to get these tests to pass. You're feeding 'expectation' into your POST and PUT requests, and 'expectation' is an array with an object inside. Without a clearer commit history I'm not sure whether the tests with this shape came first or the data.body[0] in your endpoints came first, but in any case it makes it harder to replicate the behavior outside of the tests (because more usually you'd be sending through an object that's not in an array).

The simple way to deal with this is to 1) feed an object as the .send() value for these endpoints and 2) remove that [0] from the data.body. If you want to keep the expectation in an array for testing purposes (so it's easy to refer to for the GET request, for example), you can do .send(expectation[0]) to just send the target object.